tests/support/aofmanifest.tcl

```diff
@@ -122,7 +122,7 @@ proc assert_aof_manifest_content {manifest_path content} {
     assert_equal [llength $lines] [llength $content]
 
     for { set i 0 } { $i < [llength $lines] } {incr i} {
-        assert_equal [lindex $lines $i] [lindex $content $i]
+        assert {[string first [lindex $content $i] [lindex $lines $i]] != -1}
```

Comment:
can you explain the change of this test?

Reply:
in new version, for INCR AOF, we always add `startoffset`, as bellow, but before, tcl test cases don't have this info, so they failed when useing `equal`
`file appendonly.aof.2.incr.aof seq 2 type i startoffset 224`

I even feel that we should have used `string first` from the beginning, since for AOF manifest format, we allow add more fields during the design phase.